concrete_datastore/routes/urls.py

```diff
@@ -44,6 +43,10 @@
     api_v1_1_urls,
     *swagger_urls,
 ]
+if settings.ADMIN_URL_ENABLED:
+    urlpatterns += [
+        re_path(r'^concrete-datastore-admin/', admin_site.urls, name='admin')
```

Comment:
I would take the opportunity to make this path configurable : 

```suggestion
        re_path(rf'^{settings.ADMIN_ROOT_URI}/', admin_site.urls, name='admin')
```
with

```python
ADMIN_ROOT_URI = "concrete-datastore-admin"
```